File: Java/1069.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        
        char grade = sc.next().charAt(0);
        
        sc.close();
        
        switch(grade){
            case 'A': 
                System.out.println("best!!!");
                break;
            case 'B':
                System.out.println("good!!");
                break;
            case 'C':
                System.out.println("run!");
                break;
            case 'D':
                System.out.println("slowly~");
                break;
            default:
                System.out.println("what?");
                break;
        }
    }
}

File: C++/1069.cpp
#include <iostream>

using namespace std;

int main(){
    char grade;
    cin >> grade;
    
    switch(grade){
        case 'A': 
            cout << "best!!!";
            break;
        case 'B':
            cout << "good!!";
            break;
        case 'C':
            cout << "run!";
            break;
        case 'D':
            cout << "slowly~";
            break;
        default:
            cout << "what?";
    }
    return 0;
}